<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuriaSTAT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="extraTabWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>Log</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="datasetGroupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>276</width>
      <height>225</height>
     </rect>
    </property>
    <property name="title">
     <string>Dataset</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="mainPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mainEditor" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>120</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mainOutput" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>120</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionDataset"/>
    <addaction name="actionExtras"/>
    <addaction name="actionReport"/>
    <addaction name="actionShowOutput"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionData"/>
     <addaction name="actionScript_2"/>
     <addaction name="actionOpenOutput"/>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewData"/>
     <addaction name="actionScript"/>
     <addaction name="actionNewOutput"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExample_Dataset"/>
    <addaction name="actionRecent_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionOption"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionFind_and_Replace"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Console"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_Variable"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QMenu" name="menuMerge">
     <property name="title">
      <string>Merge</string>
     </property>
     <addaction name="actionMerge_by_Row"/>
     <addaction name="actionMerge_by_Column"/>
    </widget>
    <addaction name="actionShowData_Editor"/>
    <addaction name="separator"/>
    <addaction name="actionSort"/>
    <addaction name="actionTranspose"/>
    <addaction name="menuMerge"/>
    <addaction name="actionAggregate"/>
    <addaction name="separator"/>
    <addaction name="actionSplit"/>
    <addaction name="actionIdentify_Duplicate_Cases"/>
    <addaction name="actionDuplicate_Count"/>
    <addaction name="actionSelect"/>
    <addaction name="actionWeight_Cases"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>Transform</string>
    </property>
    <widget class="QMenu" name="menuRecode">
     <property name="title">
      <string>Recode</string>
     </property>
     <addaction name="actionRecodeSameVar"/>
     <addaction name="actionRecodeDiffVar"/>
    </widget>
    <addaction name="actionCompute_Variable"/>
    <addaction name="separator"/>
    <addaction name="menuRecode"/>
    <addaction name="actionAutomatic_Recode"/>
    <addaction name="actionRandom_Number_Generator"/>
    <addaction name="separator"/>
    <addaction name="actionReplace_Missing_Value"/>
    <addaction name="separator"/>
    <addaction name="actionRank_Case"/>
   </widget>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistics</string>
    </property>
    <widget class="QMenu" name="menuDimension_Reduction">
     <property name="title">
      <string>Dimension Reduction</string>
     </property>
     <addaction name="actionPrincipal_Component_Analysis"/>
     <addaction name="actionExploratory_Factor_Analysis"/>
     <addaction name="actionCorrespondence_Analysis"/>
     <addaction name="actionOptimal_Scaling"/>
    </widget>
    <widget class="QMenu" name="menuRegression">
     <property name="title">
      <string>Regression</string>
     </property>
     <addaction name="actionLinear_Regression"/>
     <addaction name="actionNonlinear_Regression"/>
     <addaction name="separator"/>
     <addaction name="actionLogistic_Regression"/>
    </widget>
    <widget class="QMenu" name="menuGeneralized_Linear_Model">
     <property name="title">
      <string>Generalized Linear Model</string>
     </property>
     <addaction name="actionGeneralized_linear_model"/>
     <addaction name="actionGeneralized_Estimating_Equations"/>
    </widget>
    <widget class="QMenu" name="menuCompare_Means">
     <property name="title">
      <string>Compare Means</string>
     </property>
     <addaction name="actionMeans"/>
     <addaction name="actionOne_Sample_T_test"/>
    </widget>
    <widget class="QMenu" name="menuClassification">
     <property name="title">
      <string>Classification</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionDiscriminant_Analysis"/>
     <addaction name="actionK_Nearest_Neighbor"/>
     <addaction name="actionSupport_Vector_Machine"/>
    </widget>
    <widget class="QMenu" name="menuScale_Analysis">
     <property name="title">
      <string>Scale Analysis</string>
     </property>
     <addaction name="actionReliabilyti_Analysis"/>
     <addaction name="actionMultidimensional_Scaling"/>
    </widget>
    <widget class="QMenu" name="menuCluster_Analysis">
     <property name="title">
      <string>Cluster Analysis</string>
     </property>
     <addaction name="actionK_Means_Cluster"/>
     <addaction name="actionHieararchical_Cluster"/>
     <addaction name="actionTree_Analysis"/>
    </widget>
    <addaction name="actionDescriptive"/>
    <addaction name="menuCompare_Means"/>
    <addaction name="menuGeneralized_Linear_Model"/>
    <addaction name="actionCorrelation"/>
    <addaction name="menuRegression"/>
    <addaction name="menuClassification"/>
    <addaction name="menuCluster_Analysis"/>
    <addaction name="menuDimension_Reduction"/>
    <addaction name="menuScale_Analysis"/>
    <addaction name="actionTime_Series"/>
    <addaction name="separator"/>
    <addaction name="actionNonparametric"/>
   </widget>
   <widget class="QMenu" name="menuGraphics">
    <property name="title">
     <string>Graphics</string>
    </property>
    <addaction name="actionScatter_Plot"/>
    <addaction name="actionBuble_Plot"/>
    <addaction name="actionSunflower_Plot"/>
    <addaction name="actionMatrix_of_Scatter_Plot"/>
    <addaction name="actionConditioning_Plot"/>
    <addaction name="separator"/>
    <addaction name="actionBar_Plot"/>
    <addaction name="actionDot_Plot"/>
    <addaction name="separator"/>
    <addaction name="actionBox_Plot"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionDash_LIne"/>
    <addaction name="actionSteam_and_leaf_Plot"/>
    <addaction name="action3_D_Plot"/>
    <addaction name="actionCircular_Plot"/>
    <addaction name="separator"/>
    <addaction name="actionReceiver_Operation_Characteristic_ROC_Curve"/>
    <addaction name="actionKaplan_Meier_Plot"/>
    <addaction name="actionPlot_an_arbitary_function_plot"/>
    <addaction name="actionEmpirical_Cumulatice_Probability_Density_Plot"/>
    <addaction name="actionEmpirical_Probability_Density_Plot"/>
    <addaction name="actionNormal_Qunatile_Quantile_Plot"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTopics"/>
    <addaction name="actionTutorial"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuData"/>
   <addaction name="menuTransform"/>
   <addaction name="menuGraphics"/>
   <addaction name="menuStatistic"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowHome"/>
   <addaction name="actionShowData_Editor"/>
   <addaction name="actionShowOutput"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDataset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dataset</string>
   </property>
  </action>
  <action name="actionExtras">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Extras</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>Data...</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionNewOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionScript_2">
   <property name="text">
    <string>Script...</string>
   </property>
  </action>
  <action name="actionOpenOutput">
   <property name="text">
    <string>Output...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExample_Dataset">
   <property name="text">
    <string>Example Data...</string>
   </property>
  </action>
  <action name="actionRecent_Dataset">
   <property name="text">
    <string>Recent Data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind_and_Replace">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionClear_Console">
   <property name="text">
    <string>Clear Console</string>
   </property>
  </action>
  <action name="actionGo_to_Variable">
   <property name="text">
    <string>Go to Variable</string>
   </property>
  </action>
  <action name="actionIdentify_Duplicate_Cases">
   <property name="text">
    <string>Identify Duplicate Cases</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
  <action name="actionAggregate">
   <property name="text">
    <string>Aggregate</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select Cases</string>
   </property>
  </action>
  <action name="actionSplit">
   <property name="text">
    <string>Split</string>
   </property>
  </action>
  <action name="actionCases">
   <property name="text">
    <string>Cases</string>
   </property>
  </action>
  <action name="actionCompute_Variable">
   <property name="text">
    <string>Compute Variable...</string>
   </property>
  </action>
  <action name="actionAutomatic_Recode">
   <property name="text">
    <string>Automatic Recode...</string>
   </property>
  </action>
  <action name="actionVisual_Binning">
   <property name="text">
    <string>Visual Binning</string>
   </property>
  </action>
  <action name="actionOptimal_Binning">
   <property name="text">
    <string>Optimal Binning</string>
   </property>
  </action>
  <action name="actionRandom_Number_Generator">
   <property name="text">
    <string>Random Number Generator...</string>
   </property>
  </action>
  <action name="actionReplace_Missing_Value">
   <property name="text">
    <string>Replace Missing Value...</string>
   </property>
  </action>
  <action name="actionRank_Case">
   <property name="text">
    <string>Rank Case...</string>
   </property>
  </action>
  <action name="actionRecodeSameVar">
   <property name="text">
    <string>Same Variable...</string>
   </property>
  </action>
  <action name="actionRecodeDiffVar">
   <property name="text">
    <string>Different Variable...</string>
   </property>
  </action>
  <action name="actionScatter_Plot">
   <property name="text">
    <string>Scatter Plot</string>
   </property>
  </action>
  <action name="actionBuble_Plot">
   <property name="text">
    <string>Buble Plot</string>
   </property>
  </action>
  <action name="actionSunflower_Plot">
   <property name="text">
    <string>Sunflower Plot</string>
   </property>
  </action>
  <action name="actionMatrix_of_Scatter_Plot">
   <property name="text">
    <string>Matrix of Scatter Plot</string>
   </property>
  </action>
  <action name="actionConditioning_Plot">
   <property name="text">
    <string>Conditioning Plot</string>
   </property>
  </action>
  <action name="actionBar_Plot">
   <property name="text">
    <string>Bar Plot</string>
   </property>
  </action>
  <action name="actionDot_Plot">
   <property name="text">
    <string>Dot Plot</string>
   </property>
  </action>
  <action name="actionBox_Plot">
   <property name="text">
    <string>Box Plot</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionDash_LIne">
   <property name="text">
    <string>Dash LIne</string>
   </property>
  </action>
  <action name="actionSteam_and_leaf_Plot">
   <property name="text">
    <string>Steam-and-leaf Plot</string>
   </property>
  </action>
  <action name="action3_D_Plot">
   <property name="text">
    <string>3-D Plot</string>
   </property>
  </action>
  <action name="actionCircular_Plot">
   <property name="text">
    <string>Circular Plot</string>
   </property>
  </action>
  <action name="actionReceiver_Operation_Characteristic_ROC_Curve">
   <property name="text">
    <string>Receiver Operation Characteristic (ROC) Curve</string>
   </property>
  </action>
  <action name="actionKaplan_Meier_Plot">
   <property name="text">
    <string>Kaplan-Meier Plot</string>
   </property>
  </action>
  <action name="actionPlot_an_arbitary_function_plot">
   <property name="text">
    <string>Plot an arbitary function plot</string>
   </property>
  </action>
  <action name="actionEmpirical_Cumulatice_Probability_Density_Plot">
   <property name="text">
    <string>Empirical Cumulatice Probability Density Plot</string>
   </property>
  </action>
  <action name="actionEmpirical_Probability_Density_Plot">
   <property name="text">
    <string>Empirical Probability Density Plot</string>
   </property>
  </action>
  <action name="actionNormal_Qunatile_Quantile_Plot">
   <property name="text">
    <string>Normal Qunatile-Quantile Plot</string>
   </property>
  </action>
  <action name="actionTopics">
   <property name="text">
    <string>Topics</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionNewData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionWeight_Cases">
   <property name="text">
    <string>Weight Cases</string>
   </property>
  </action>
  <action name="actionDescriptive">
   <property name="text">
    <string>Descriptive...</string>
   </property>
  </action>
  <action name="actionNonparametric">
   <property name="text">
    <string>Nonparametric</string>
   </property>
  </action>
  <action name="actionTime_Series">
   <property name="text">
    <string>Time Series</string>
   </property>
  </action>
  <action name="actionPrincipal_Component_Analysis">
   <property name="text">
    <string>Principal Component Analysis</string>
   </property>
  </action>
  <action name="actionExploratory_Factor_Analysis">
   <property name="text">
    <string>Exploratory Factor Analysis</string>
   </property>
  </action>
  <action name="actionCorrelation">
   <property name="text">
    <string>Correlation</string>
   </property>
  </action>
  <action name="actionMeans">
   <property name="text">
    <string>Means</string>
   </property>
  </action>
  <action name="actionOne_Sample_T_test">
   <property name="text">
    <string>One Sample T-test</string>
   </property>
  </action>
  <action name="actionCluster">
   <property name="text">
    <string>K-Means Cluster</string>
   </property>
  </action>
  <action name="actionHieararchical_Cluster">
   <property name="text">
    <string>Hierarchical Cluster</string>
   </property>
  </action>
  <action name="actionCorrespondence_Analysis">
   <property name="text">
    <string>Correspondence Analysis</string>
   </property>
  </action>
  <action name="actionOptimal_Scaling">
   <property name="text">
    <string>Optimal Scaling</string>
   </property>
  </action>
  <action name="actionTree_Analysis">
   <property name="text">
    <string>Tree Analysis</string>
   </property>
  </action>
  <action name="actionDiscriminant_Analysis">
   <property name="text">
    <string>Discriminant Analysis</string>
   </property>
  </action>
  <action name="actionReliabilyti_Analysis">
   <property name="text">
    <string>Reliability Analysis</string>
   </property>
  </action>
  <action name="actionMultidimensional_Scaling">
   <property name="text">
    <string>Multidimensional Scaling</string>
   </property>
  </action>
  <action name="actionK_Nearest_Neighbor">
   <property name="text">
    <string>K-Nearest Neighbor</string>
   </property>
  </action>
  <action name="actionSupport_Vector_Machine">
   <property name="text">
    <string>Support Vector Machine</string>
   </property>
  </action>
  <action name="actionK_Means_Cluster">
   <property name="text">
    <string>K-Means Cluster</string>
   </property>
  </action>
  <action name="actionLinear_Regression">
   <property name="text">
    <string>Linear Regression</string>
   </property>
  </action>
  <action name="actionLogistic_Regression">
   <property name="text">
    <string>Logistic Regression</string>
   </property>
  </action>
  <action name="actionNonlinear_Regression">
   <property name="text">
    <string>Nonlinear Regression</string>
   </property>
  </action>
  <action name="actionGeneralized_linear_model">
   <property name="text">
    <string>Generalized Linear Models</string>
   </property>
  </action>
  <action name="actionGeneralized_Estimating_Equations">
   <property name="text">
    <string>Generalized Estimating Equations</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="actionMerge_by_Row">
   <property name="text">
    <string>Merge by Row</string>
   </property>
  </action>
  <action name="actionMerge_by_Column">
   <property name="text">
    <string>Merge by Column</string>
   </property>
  </action>
  <action name="actionShowOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="toolTip">
    <string>Show Output</string>
   </property>
  </action>
  <action name="actionShowHome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Show Home</string>
   </property>
  </action>
  <action name="actionShowData_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Editor</string>
   </property>
   <property name="toolTip">
    <string>Show Data Editor</string>
   </property>
  </action>
  <action name="actionDuplicate_Count">
   <property name="text">
    <string>Duplicate Count</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
